library/src/main/scala/scala/scalajs/js/typedarray/BigUint64Array.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package scala.scalajs.js.typedarray
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation._
+
+/** <span class="badge badge-ecma2019" style="float: right;">ECMAScript 2019</span>
+ *
+ *  A [[TypedArray]] of unsigned [[js.BigInt]]
+ */
+@js.native
+@JSGlobal
+class BigUint64Array private[this] ()
+    extends TypedArray[js.BigInt, BigUint64Array] {
+
+  /** Constructs a BigInt64Array with the given length. Initialized to all 0 */
```

Comment:
```suggestion
  /** Constructs a BigUint64Array with the given length. Initialized to all 0 */
```
Same in the other constructors below.